lang/python/python-evdev/Makefile

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=python-evdev
+PKG_VERSION:=0.7.0
+PKG_RELEASE:=1
+
+PKG_LICENSE:=BSD-3-Clause
+PKG_MAINTAINER:=Paulo Costa <me@paulo.costa.nom.br>
```

Comment:
regarding your question
> I honestly haven't been keeping up with this, would you like to take over the PR?

I am a bit unclear about what you mean :)
Do you want me to take over this package as well ?

if you want, i can be co-maintainer (or 2nd maintainer) ;
all you need to do is add me to this line;
something like
`PKG_MAINTAINER:=Paulo Costa <me@paulo.costa.nom.br>, Alexandru Ardelean <ardeleanalex@gmail.com>`

of course, this is entirely up to you;
i am fine to be added to this [ if this is fine with you ]